src/Table.php

```diff
@@ -509,6 +509,14 @@ public function onRowClick($action): void
         $this->addClass('selectable');
         $this->js(true)->find('tbody')->css('cursor', 'pointer');
 
+        /* Do not trigger row click event if click stems from row content
+         like checkboxes, or links, marked as atk4-norowclick
+         */
+        $this->js(true)->find('a, .checkbox, .atk4-norowclick')->on(
+            'click',
+            new JsExpression('function(event) { event.stopPropagation() }')
```

Comment:
the comment does just dummily tells now what the code does/says itself - remove

`new JsExpression('function(event)` I think `JsFunction` can simplify it

also check whole PR diff/changes, `src/Table/Column/Checkbox.php` has some unreverted changes